src/camera.cpp

```diff
@@ -484,13 +484,12 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime,
 
 void Camera::updateViewingRange()
 {
+	f32 viewing_range = g_settings->getFloat("viewing_range");
+	m_draw_control.wanted_range = viewing_range;
 	if (m_draw_control.range_all) {
 		m_cameranode->setFarValue(100000.0);
 		return;
 	}
-
```

Comment:
> Is that what you are saying, i.e. with range_all _always retrieve max_block_send_distance range of blocks from the server?

No that's not what i meant, sorry.

> As it is now you'd have to select range_all and increase the view_range to get up to max_block_send_distance worth of blocks.

This is fine, as it is in this first commit.